templates/common/infra/bicep/core/security/registry-access.bicep

```diff
@@ -0,0 +1,18 @@
+param containerRegistryName string
+param principalId string
+
+var AcrPullRole = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '7f951dda-4ed3-4680-a7ca-43fe172d538d')
+
+resource aks_acr_pull 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```

Comment:
I don't think we use snake case anywhere

Reply:
Thanks - some copy/paste that I forgot to update :)